apps/frontend/src/api/__tests__/hooks.test.tsx

```diff
@@ -0,0 +1,54 @@
+import useFetch from 'use-http';
+import { useFetchOptions } from '../util';
+
+import { useApiGet } from '../hooks';
+
+jest.mock('use-http');
```

Comment:
not a big fan of this mock-heavy testing :/ this e.g. misses deleting the `return ` from the hook :D
I was hoping for a `@testing-library/react-hooks` test.
I think this may be a good topic to pair on on Monday, we haven't actually done before! :sweat_smile: